shared/actions/devices.js

```diff
@@ -41,10 +44,25 @@ export function removeDevice (deviceID: string, name: string, currentDevice: boo
   return {type: Constants.removeDevice, payload: {deviceID, name, currentDevice}}
 }
 
+export function showRemovePage (device: Device): ShowRemovePage {
+  return {type: Constants.showRemovePage, payload: {device}}
+}
+
 export function generatePaperKey (): GeneratePaperKey {
   return {type: Constants.generatePaperKey, payload: undefined}
 }
 
+function * _deviceShowRemovePageSaga (showRemovePageAction: ShowRemovePage): SagaGenerator<any, any> {
+  const device = showRemovePageAction.payload.device
+  let endangeredTLFs = {endangeredTLFs: []}
+  try {
+    endangeredTLFs = yield call(rekeyGetRevokeWarningRpcPromise, {param: {targetDevice: device.deviceID}})
```

Comment:
hmmm. I'm a little worried about this call taking awhile and the user being confused. This is fine short term but in general i think we should either
1. Do some waiting state (either make the button you clicked waiting or something else, cursor maybe? if you click a row)
or
2. Actually go to the new page and have a spinner and do this loading when the component routes.

let's discuss during scrum